### DISCUSSION TITLE
My benefits are still not available

### DISCUSSION THREAD

**Discussion by frowing37** [community/community#71596] [GitHub Education]
### Hi everyone,

"Congrats! Your request for [@frowing37](https://github.com/frowing37) was approved and your benefits will be available soon."

website show this message but my benefits page not available for 2 days. How can fix that ?

**Comment by hahuyhungdev** [comment]
you can provider detail?

**Comment by ghost** [comment]
I am facing with same issue

**Comment by yao-creative** [comment]
Same for me

**Comment by frowing37** [comment]
Update : My benefits available now, I got this error and someday after that is fixed
